frontend/src/common/entities.ts

```diff
@@ -76,6 +76,7 @@ export interface Collection {
   updated_at: number;
   has_revision: boolean;
   revision_diff: boolean;
+  tombstone?: boolean;
```

Comment:
readability: `isTomstoned`

Reply:
or is `isTombstone`. This avoid using our made up word "tombstoned"

Reply:
this is received from backend

Reply:
ahh I see, nvm!